models.js

```diff
@@ -7,21 +7,26 @@ const userSchema = new mongoose.Schema({
 
 });
 
-userSchema.pre('save', function(next){
-    let user = this;
+
+userSchema.pre('save', function(next) {
```

Comment:
when the pre save is used, it means, that instead of doing `User.create`, you'll have to create a user in a different way

```
let user = new User();
    user.email = blah blah
    user.password = blah blah
    use.save(function(){
        blah blah
});
```